ROADMAP.md

```diff
@@ -5,52 +5,46 @@ evolve and change over time. In particular the features in later versions are li
 * Making Pantheon a First Class Client
 * Istanbul Support 
-* Smart Contract Based Permissioning
-* Advanced Privacy
+* State Pruning 
+* Tracing APIs
+* Disaster recovery 
 
 ### Making Pantheon a First Class Client
 
-First and foremost, we want Pantheon to be a first class client for usage on both mainnet and permissioned networks. This entails maintaining compatibility with mainnet, providing permissioning features and constantly improving Pantheon's performance. Some recent additions to the client have been [UPnP Support](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Networking/Using-UPnP/) and a [GraphQL interface](https://docs.pantheon.pegasys.tech/en/latest/Pantheon-API/GraphQL/). 
+First and foremost, we want Pantheon to be a first class client for usage on both mainnet and permissioned networks. 
+This entails maintaining compatibility with mainnet, providing permissioning features and constantly improving Pantheon's performance. 
+Some recent additions to the client have been [UPnP Support](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Networking/Using-UPnP/)
+and a [GraphQL interface](https://docs.pantheon.pegasys.tech/en/latest/Pantheon-API/GraphQL/). 
 
 ### Istanbul Support
 
-Pantheon will support the upcomming Istanbul network upgrade and will implement all required EIPs as per the [Hardfork Meta: Istanbul EIP](https://eips.ethereum.org/EIPS/eip-1679). 
-
-### Smart Contract Based Permissioning
-
-Building on the smart contract based permissioning mechanism implemented in v1.1, additional tooling will be provided through a dapp to simplify and enhance the interaction with the smart contracts. Permissioning for Ethereum accounts will also be introduced.
-
-### Advanced Privacy
-The current privacy functionality will be enhanced as described in the [privacy roadmap](PRIVACYROADMAP.MD).
-
-## Next (v1.3/1.4)
-The key areas for next are:
-* State Pruning
-* Secure key-store and key management
-* Disaster Recovery
-* Permissioning using RBAC
-* IBFT 2.x 
-* Ethereum 1.x 
+Pantheon will support the upcoming Istanbul network upgrade and implement all required EIPs as per the [Hardfork Meta: Istanbul EIP](https://eips.ethereum.org/EIPS/eip-1679). 
 
 ### State Pruning 
 
 State pruning will be implemented. State pruning reduces the disk space required for the Pantheon database by discarding outdated world state data. 
 
-### Secure Key-Store and Key Management
+### Tracing APIs 
 
-Pantheon will enable external key management for encryption and decryption operations, ensuring that key management is not solely managed by an external party. 
+Additional tracing APIs to be added. 
 
 ### Disaster Recovery
 
 Support key-value storage in relational databases to solidify a robust Disaster Recovery process. Note: Orion to support Oracle and Postgres in 1.3. 
 
-### Permissioning using RBAC
+## Next (v1.4)
+The key areas for next are:
+* Privacy group modification 
+* IBFT 2.x 
```

Comment:
Maybe instead we have:
- Enhancing key management capabilities
- Migration tools

Reply:
Done